gluten-core/src/main/scala/io/glutenproject/execution/BatchScanExecTransformer.scala

```diff
@@ -119,15 +117,20 @@ class BatchScanExecTransformer(
   @transient protected lazy val filteredFlattenPartitions: Seq[InputPartition] =
     filteredPartitions.flatten
 
-  private def getPushdownFilters: Seq[Expression] = {
-    pushdownFilters
-  }
-
   @transient override lazy val fileFormat: ReadFileFormat = scan.getClass.getSimpleName match {
     case "OrcScan" => ReadFileFormat.OrcReadFormat
     case "ParquetScan" => ReadFileFormat.ParquetReadFormat
     case "DwrfScan" => ReadFileFormat.DwrfReadFormat
     case "ClickHouseScan" => ReadFileFormat.MergeTreeReadFormat
     case _ => ReadFileFormat.UnknownFormat
   }
+
+  override def doCanonicalize(): BatchScanExecTransformer = {
+    val canonicalized = super.doCanonicalize()
```

Comment:
Just wondering whether we can directly use the members of `BatchScanExecTransformer` to doCanonicalize. There may have some differences. Thanks!

Reply:
BatchScanExecTransformer inherits BatchScanExec, and their parameters should be exactly the same.